builder/src/runtime.rs

```diff
@@ -466,7 +483,10 @@ MEMORY
     prog (rx) : ORIGIN = $APPS_OFFSET, LENGTH = $APPS_SIZE
     ram (rwx) : ORIGIN = $DATA_RAM_OFFSET, LENGTH = $DATA_RAM_SIZE
     dccm (rw) : ORIGIN = $DCCM_OFFSET, LENGTH = $DCCM_SIZE
+    flash (r) : ORIGIN = $FLASH_OFFSET, LENGTH = $FLASH_SIZE
 }
 
+PAGE_SIZE = 256;
```

Comment:
This might be good to pass in as a parameter to the builder, or part of the config for the platform.